server/src/test/java/org/elasticsearch/tasks/TaskInfoTests.java

```diff
@@ -227,13 +233,16 @@ protected TaskInfo mutateInstance(TaskInfo info) {
     }
 
     static TaskInfo randomTaskInfo() {
+        return randomTaskInfo(randomLong());
```

Comment:
I think randomNonNegativeLong() is reasonable here as well.

Reply:
cool.